frontend/src/components/commons/PopupBox.jsx

```diff
@@ -102,56 +90,52 @@ class PopupBox extends Component {
       rightButtonState
     } = this.props;
 
+    // If a root node exists, the app is being served, otherwise it's a unit test.
+    let ariaHideApp = true;
+    if (document.getElementById("#root")) {
+      Modal.setAppElement("#root");
+    } else {
+      // Unit tests do not consider outside of the dialog.
+      ariaHideApp = false;
+    }
     return (
       <Modal
-        show={show}
-        onHide={handleClose}
-        autoFocus={true}
-        enforceFocus={true}
-        aria-hidden={false}
-        backdrop={this.props.isBackdropStatic ? false : true}
+        isOpen={show}
+        onRequestClose={handleClose}
+        style={customStyles}
+        contentLabel={title}
+        shouldCloseOnOverlayClick={!this.props.isBackdropStatic}
+        aria={{
+          labelledby: "modal-heading",
+          describedby: "modal-description"
```

Comment:
You would need to put these under localize for translation.

Reply:
Actually, since these are labeldby and describeby, would she? title and description are passed in through the props.

Reply:
These aria fields simply link to the html element that contains the description. They are never read out to a screen reader as Michael pointed out.